docs/adr/0005-withdraw-expiry.rst

```diff
@@ -0,0 +1,91 @@
+Withdraw & Expiry
```

Comment:
I think this is a duplicate of the above.

Reply:
No, the above one is the duplicate.

Reply:
Ahh yes,,, 0004 should be removed.

Reply:
yeah. What's this file for? It even has the counter incremented to 5 :smile: